apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.TableCrud.cs

```diff
@@ -235,11 +235,86 @@ public abstract partial class BigQueryClient
         #region CreateTable (autogenerated - do not edit manually)
         /// <summary>
         /// Creates the specified table within this client's project.
+        /// This method just creates a <see cref="TableReference"/> and delegates to <see cref="CreateTable(TableReference, Table, CreateTableOptions)"/>.
+        /// </summary>
+        /// <param name="datasetId">The dataset ID. Must not be null.</param>
+        /// <param name="tableId">The table ID. Must not be null.</param>
+        /// <param name="resource">The table resource representation to use for the creation. May be null in which case default values will be used. If this table's <see cref="Table.TableReference"/> is specified, then it must be the same as the one obtained from the other parameters.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>The newly created table.</returns>
+        public virtual BigQueryTable CreateTable(string datasetId, string tableId, Table resource = null, CreateTableOptions options = null) =>
```

Comment:
I don't think I'd make the table optional here - I think "I want to create a table with no schema" is so rarely useful that we force people to spell that out. (I would make it a required parameter and validate that it's not null.)

Reply:
Agreed offline to not allow null table or schema in the corresponding overloads. On it now.